TweetBook/Installers/SwaggerInstaller.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.Filters;
+using System.Reflection;
+
+namespace TweetBook.Installers
+{
+    public class SwaggerInstaller : IInstaller
+    {
+        public void InstallServices(WebApplicationBuilder builder)
+        {
+            builder.Services.AddSwaggerGen(x =>
+            {
+                var security = new Dictionary<string, IEnumerable<string>>
+    {
+        {"Bearer",new string[0]}
+    };
```

Comment:
Something is weird with the file formatting since some of the parenthesis are not aligned,